packages/gatsby/cache-dir/loader.js

```diff
@@ -573,15 +573,19 @@ const createComponentUrls = componentChunkName =>
 
 export class ProdLoader extends BaseLoader {
   constructor(asyncRequires, matchPaths, pageData) {
-    const loadComponent = chunkName => {
-      if (!asyncRequires.components[chunkName]) {
+    const loadComponent = (chunkName, exportType = `components`) => {
+      if (!global.hasPartialHydration) {
```

Comment:
is `global.hasPartialHydration` being somewhere? I don't see it anywhere in existing code in `master` (yet)

Reply:
This will be available with partialHydration https://github.com/gatsbyjs/gatsby/pull/36485